docs/integration/reonboarding.adoc

```diff
@@ -14,17 +14,17 @@ Internally, the endpoint gets the same identification (deviceAlternateId, sensor
 
 [IMPORTANT]
 ====
-Reonboarding is major as certificates are only valid for currently *one year*
+Reonboarding is very important as certificates are only valid for currently *one year*
```

Comment:
By "major" I mean "major due to certification". But we need to update this as reonboarding is only major to those endpoints that do not use router devices.

Reply:
Hmm... I think I don't get what you want to say... what is the German term for what you want to express?

Reply:
I think "required for CUs" is the correct term. German word would be something like "zwingend erforderlich".

Reply:
"It is vital when not using RouterDevices ..." and I'm fine ;)

Reply:
Done